custom-metrics-stackdriver-adapter/deploy/staging/adapter_new_resource_model_with_core_metrics.yaml

```diff
@@ -0,0 +1,252 @@
+# A staging deployment of Custom Metrics - Stackdriver Adapter.
```

Comment:
The staging file was created for OSS e2e tests in mind, I think it will be more appropriate to create another directory like "test" or "dev" for experimental deployments.